polkadot-protocol/basics/randomness.md

```diff
@@ -1,4 +1,76 @@
 ---
 title: Randomness
 description: TODO
----
\ No newline at end of file
+---
+
+# Randomness
+
+Randomness is crucial in **Proof of Stake** blockchains to ensure a fair and unpredictable distribution of validator duties. 
+
+However, computers are inherently deterministic, meaning the same input always produces the same output. What we typically refer to as "random" numbers on a computer are actually pseudo-random. 
+
+These numbers rely on an initial "seed", which can come from external sources like [atmospheric noise](https://www.random.org/randomness/){target=\_blank}, [heart rates](https://mdpi.altmetric.com/details/47574324){target=\_blank}, or even [lava lamps](https://en.wikipedia.org/wiki/Lavarand){target=\_blank}. While this may seem random, given the same "seed", the same sequence of numbers will always be generated.
+
+In a global blockchain network, relying on real-world entropy for randomness isn’t feasible because these inputs vary by time and location. If nodes use different inputs, blockchains can fork. Hence, real-world randomness isn't suitable for use as a seed in blockchain systems.
+
+Currently, two primary methods for generating randomness in blockchains are used: `RANDAO` and `VRF` (Verifiable Random Function). Polkadot adopts the `VRF` approach for its randomness.
+
+## VRF
+
+A **Verifiable Random Function (VRF)** is a cryptographic function that generates a random number along with a proof that ensures the number was produced by the submitter. This proof allows anyone to verify the validity of the random number.
+
+Polkadot's VRF is similar to the one used in [**Ouroboros Praos**](https://eprint.iacr.org/2017/573.pdf){target=\_blank}, which secures randomness for block production in systems like [BABE](link to BABE page) (Polkadot’s block production mechanism). 
```

Comment:
I suggest using the flag `TODO:update-path` when we need to add a reference to a page that does not exist yet. This way, we can do `Ctrl+F` through the whole repo and find those occurrences easily :)

```suggestion
Polkadot's VRF is similar to the one used in [**Ouroboros Praos**](https://eprint.iacr.org/2017/573.pdf){target=\_blank}, which secures randomness for block production in systems like [BABE](TODO:update-path){target=\_blank} (Polkadot’s block production mechanism). 
```